src/Tribe/Repositories/Event.php

```diff
@@ -1680,9 +1701,12 @@ protected function order_by_venue( $after = false ) {
 		);
 
 		$filter_id = 'order_by_venue';
-		$order     = Arr::get_in_any( [ $this->query_args, $this->default_args ], 'order', 'ASC' );
-		$this->filter_query->orderby( [ 'venue' => $order ], $filter_id, true, $after );
-		$this->filter_query->fields( "{$posts_table}.post_title AS venue", $filter_id, true );
+		$order     = $order === null
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 9e8ca05fff9f0ad74244a2a8c1c7d0a4</sub>